Source/DafnyRuntime/DafnyRuntime.py

```diff
@@ -0,0 +1,4 @@
+class _dafny:
```

Comment:
Do you need this level of encapsulation (class + instance)?  You could use static methods, or even define functions directly inside DafnyRuntime?